extensions/sdk/src/Rally.ts

```diff
@@ -99,9 +99,49 @@ export class Rally {
       connectFirestoreEmulator(this._db, 'localhost', 8080);
     }
 
+    this.storeAttributionCodes()
+      .catch((err) => console.error("Could not store attribution codes:", err));
```

Comment:
I think we try to do this within reason, this is just a top-level catch-all (there isn't much we can really do since it's called from the constructor, besides just throw an error). Do you think there shouldn't be a `catch` here at all?